Chapter11/booktracker/application/booktracker/blueprints/frontend/reload.py

```diff
@@ -63,24 +67,26 @@ async def runner(app: Sanic, app_server):
         app.is_stopping = True
 
 
-def setup_livereload(bp: Blueprint):
+def setup_livereload(bp: Blueprint) -> None:
+    global livereload
     livereload.ctx.bp = bp
 
     @bp.before_server_start
-    async def start(app, _):
+    async def start(app: Sanic, _: Any) -> None:
```

Comment:
It's not `Any`. This is the loop, but since it is not needed I like to use `_`. A proper type annotation would be `AbstractBaseLoop`, but I think that is more distracting than anything.

```suggestion
    async def start(app: Sanic, _) -> None:
```